minifier/index.d.ts

```diff
@@ -0,0 +1,82 @@
+// Type definitions for minifier v0.8.0
+// Project: https://github.com/fizker/minifier
+// Definitions by: Jeongho Nam <http://samchon.org>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module "minifier"
```

Comment:
do not use ambient module declarations. instead, export from the file directelly, e.g.

export declare function minify(source: string): void;